ui/ui-components/components/navigation/setupStepsNavProgressBar.tsx

```diff
@@ -58,36 +59,52 @@ export default function SetupStepsNavProgressBar({
 
   if (!shouldDisplay) return null;
 
+  if (isOnBoardingComplete) {
+    return (
+      <Row className="pt-1 px-4">
+        <Col>
+          <Link href="/setup">
+            <a>Get Started:</a>
+          </Link>
+        </Col>
+      </Row>
+    );
+  }
```

Comment:
Maybe the screenshots don't match this code - I think this says that if you are complete, you'll always see a link with `Get Started:` (with a ":") and no 🎉?

Reply:
Ooh.  Yep.  Missed a file in a commit.  This code is definitely wrong.

Reply:
Ohh, yeah.  It looks like I checked an incorrect version in.